NetKet/Machine/ffnn.hpp

```diff
@@ -56,7 +56,7 @@ class FFNN : public AbstractMachine<T> {
   using LookupType = typename AbstractMachine<T>::LookupType;
 
   explicit FFNN(const AbstractHilbert &hilbert,
-                const std::vector<AbstractLayer<T> *> &layers)
+                std::vector<AbstractLayer<T> *> layers)
       : hilbert_(hilbert), layers_(layers), nv_(hilbert.Size()) {
```

Comment:
I think you forgot to call `std::move` here. It should be `layers_(std::move(layers))`.

Reply:
True, I changed that now